plane/plane-tests/tests/common/test_env.rs

```diff
@@ -16,8 +16,9 @@ use plane::{
 use std::{
     net::{IpAddr, Ipv4Addr},
     path::{Path, PathBuf},
-    sync::{Arc, Mutex},
+    sync::Arc,
 };
+use tokio::sync::Mutex;
```

Comment:
switching the import makes all mutexes async, but it looks like only `db_lock` is held across an await, is that right?

Reply:
confirmed, let's just use `tokio::sync::Mutex` on 30 and keep the standard mutex import for the rest.

Reply:
Fixed